api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ContractsApiControllerTest.java

```diff
@@ -84,17 +92,23 @@ public void voteOnContract_validInput_returnsOk() {
     @Test
     public void voteOnContract_validInputAndVotingIsComplete_settlesContractPaymentsAndReturnsOk() {
```

Comment:
nit: test should be renamed to voteOnContract_validInputAndVotingIsComplete_setsContractPaymentsAndReturnsOk to reflect the updated test logic